### PR TITLE
Qwen vl infer

### PR DESCRIPTION
**Pull request by LokeZhou**
将按comment修改后重提pr

### CODE REVIEW

File: paddlemix/examples/qwen_vl/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- tiktoken
+> 注：请确保安装了以上依赖，否则无法运行。同时，需要安装[此目录](https://github.com/PaddlePaddle/PaddleNLP/tree/develop/model_zoo/gpt-3/external_ops)下的自定义OP, `python setup.py install`。如果安装后仍然找不到算子，需要额外设置PYTHONPATH
+
+## 2.2 动态图推理
+```bash
+# qwen-vl
+python paddlemix/examples/qwen_vl/run_predict.py \
+--input_image "https://bj.bcebos.com/v1/paddlenlp/models/community/GroundingDino/000000004505.jpg" \
+--prompt "Generate the caption in English with grounding:" \
+--dtype "float32"
```
Comment (jerrywgz) [inline paddlemix/examples/qwen_vl/README.md:26]: 提供参数文档说明

File: paddlemix/examples/qwen_vl/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## 1. 模型简介
+
+该模型是 [Qwen-VL](https://arxiv.org/pdf/2308.12966.pdf) 的 paddle 实现。
+
+
+## 2. Demo
+
+## 2.1 环境准备
+- python >= 3.8
```
Comment (jerrywgz) [inline paddlemix/examples/qwen_vl/README.md]: 现在paddlemix整体要求是python3.7 这里建议加下着重说明

File: paddlemix/models/qwen_vl/generation_utils.py [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+            match = False
+            for stop_token_seq in self.stop_words_ids:
+                if self._tokens_match(prev_input_ids_slice, stop_token_seq):
+                    match = True
+                    break
+            stopped_samples.append(match)
+        return stopped_samples
+
+
+def top_k_logits(logits, top_k=0, top_p=0.0, filter_value=-float("Inf")):
```
Comment (jerrywgz) [inline paddlemix/models/qwen_vl/generation_utils.py]: 这里面的一些函数是否可以删减，或者依赖paddlenlp

File: paddlemix/models/qwen_vl/visual.py [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+        )
+        out_4.stop_gradient = not True
+        self.positional_embedding = out_4
+        norm_layer = partial(paddle.nn.LayerNorm, epsilon=1e-06)
+        act_layer = paddle.nn.GELU
+        self.ln_pre = norm_layer(width)
+        self.transformer = TransformerBlock(
+            width, layers, heads, mlp_ratio, act_layer=act_layer, norm_layer=norm_layer
+        )
+        self.attn_pool = Resampler(
```
Comment (jerrywgz) [inline paddlemix/models/qwen_vl/visual.py:157]: 这里resampler是否可以抽离，保留原始vit部分，和clip，blip2类似
